hwdb.d/70-pointingstick.hwdb

```diff
@@ -87,7 +87,7 @@
 ##########################################
 # Generic
 ##########################################
-evdev:name:*[tT]rack[pP]oint*:*
+evdev:name:*[tT]rack[pP]oint*:dmi:bvn*
```

Comment:
This also matches the extended format. `*:dmi` will cover both `:phys:<phys>:ev:<ev>:dmi` in the extended match, and `:dmi` in the non-extended match.

I don't understand how this fix is supposed to work. Properties are additive. IIUC (please correct me if I got something wrong): before 57bb707d48131f4daad2b1b746eab586eb66b4f3, we query the hwdb with three patterns, and after that, we query it with four patterns. The matches returned are summed, with later additions overriding.

Please explain.